src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -302,4 +326,53 @@ private void wmc_log(String message, Integer level) {
 				break;
 		}
 	}
+	//Mojang api check
+	private boolean AuCh(Player p) {

```

Comment:
Can we change this to be a self documenting method name and document the method's signature? Maybe name it something like `isPaidUser`